tools/webpack-loaders/svg-loader/index.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+
+module.exports = function inlineSvgLoader(content) {
+    // eslint-disable-next-line no-unused-expressions
+    this.cacheable && this.cacheable(); // webpack-specfic weirdness
```

Comment:
Can you remember where you read this? Docs say loaders are [not cacheable by default](https://webpack.js.org/api/loaders/#cacheable).